### ISSUE TITLE
Open pdf in new tab instead of download

### ISSUE THREAD

**Issue by lilianabounegru** [PublicDataLab/fake-news-website#13]
Would it be possible to have the 'take me to the field guide' button open the pdf in a new tab instead of downloading it? That would mean that the pdf would have a link and it could be indexed by search engines directly and make it easier for people to find it.

**Comment by jwyg** [comment]
I've also had other comments that some people are still not finding the download link (and emailing to request PDF).

In addition to @lilianabounegru's suggestion above, perhaps we could also add direct link (as per [this ticket](https://github.com/PublicDataLab/fake-news-field-guide/issues/8)) and short link to the pop-up window?

Full link: https://github.com/PublicDataLab/fake-news-field-guide/releases/download/1.0.0/A_Field_Guide_to_Fake_News_and_other_Information_Disorders-single_pages.pdf
Short link: http://bit.ly/field-guide-pdf

@mikima - what do you think?

**Comment by mikima** [comment]
When you have a link to a PDF, [it seems to be a local setting of your browser](https://apemobile.zendesk.com/hc/en-us/articles/115001569212-PDF-files-are-downloading-instead-of-displaying-in-web-browser-or-opening-in-a-new-tab) if it will be opened in a new window or downloaded.

If you prefer, we can put a link to Zenodo. What do you think?

**Comment by jwyg** [comment]
Yes good idea. Perhaps you could add both link to Zotero and directly PDF download link? More important to have clear download links than the email signup form, and we've had feedback that some people are still not finding the link!
